### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by johnkim76**

### CODE REVIEW

**Comment by jmrodri** [comment]
Is that the new name? **Ansible Service Broker for OpenShift**?

**Comment by johnkim76** [comment]
@jmrodri I just copied & pasted (re-ordered) from a paragraph.  I will correct it.

**Comment by jwmatthews** [comment]
Please add a link for docs/README.md in the main README.md, let's specifically call out there are docs present that someone may want to read.

**Comment by jmrodri** [comment]
@johnkim76 ACk pending @jwmatthews request for change.

**Comment by johnkim76** [comment]
@jwmatthews @jmrodri , I've added the links on the main README.md

File: docs/README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
-* [Design](design.md)
+* Ansible Service Broker
+  * [Introduction](introduction.md)
+  * [Design](design.md)
+* Ansible Playbook Bundle
+  * [Design](https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/design.md)
+* Bind Operation
+  * [Data Transmission](bind-data-transmission.md)
+  * [Use Cases](usecases.md)
+* [Pagenation](pagination.md)
```
Comment (jmrodri) [inline docs/README.md]: typo: Pagenation -> Pagination

**Comment by johnkim76** [comment]
updated

**Pull request merged**